doc/releases/release-notes-3.5.rst

```diff
@@ -237,6 +237,14 @@ USB
 Devicetree
 **********
 
+* ``zephyr,memory-region-mpu`` was renamed ``zephyr,memory-attr``
```

Comment:
I think this will get covered by the bindings changes autogenerator script

Reply:
Do we have that ?? How does that work?

Reply:
https://github.com/zephyrproject-rtos/zephyr/blob/main/scripts/release/list_devicetree_bindings_changes.py courtesy of @mbolivar

Reply:
I'm going to keep this after all.